### PR TITLE
Lps 98025

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/LPS-98025

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CommerceCatalogLocalServiceImpl.java [hunk 1]
```diff
@@ -180,7 +180,8 @@ public void deleteCommerceCatalogs(long companyId) throws PortalException {
 			commerceCatalogPersistence.findByC_S(companyId, false);
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CommerceCatalogLocalServiceImpl.java:180]: look for every catalogs not just the non system

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**